companies/junyanwang/A Frozen SingAlong Celebration.php

```diff
@@ -0,0 +1,97 @@
+<?php
+   $path = "../../";
+   require_once "$path/cookies.php";
+   set_cookies(basename(__FILE__, '.php'));
+   add_product_view_count(basename(__FILE__, '.php'));
+?>
+
+<!DOCTYPE html>
+<!--[if lt IE 8 ]><html class="no-js ie ie7" lang="en"> <![endif]-->
+<!--[if IE 8 ]><html class="no-js ie ie8" lang="en"> <![endif]-->
+<!--[if IE 9 ]><html class="no-js ie ie9" lang="en"> <![endif]-->
+<!--[if (gte IE 9)|!(IE)]><!--><html class="no-js" lang="en"> <!--<![endif]-->
+
+<!--- Basic Head
+================================================== -->
+<?php include "$path/productHead.php"; ?>
+
+<body>
+
+   <!-- Header
+   =================================================== -->
+   <?php include "$path/productHeader.php"; ?>
+
+   <!-- Page Title
+   ================================================== -->
+   <section id="page-title">	
+   	  
+		<div class="row">
+
+			<div class="twelve columns">
+
+				<h1>Kun Su<span></span></h1>
```

Comment:
let's change the page title to your own name and tags for all 10 pages.

Reply:
Updated.